cddl/integrity-registers.cddl

```diff
@@ -0,0 +1,5 @@
+integrity-register-id = uint / text
```

Comment:
Should not this be 
integrity-register-id-type-choice as per the convention:
https://ietf-rats-wg.github.io/draft-ietf-rats-corim/draft-ietf-rats-corim.html#name-cddl-typographical-conventi

Reply:
done in [425d5f4](https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/189/commits/425d5f4189b32cb0876c315abb2912468aac041e)